config/configgrpc/configgrpc_test.go

```diff
@@ -117,19 +115,12 @@ func TestOptionsToConfig(t *testing.T) {
 	tests := []struct {
 		name        string
 		options     TLSConfig
-		fakeSysPool bool
 		expectError string
 	}{
 		{
 			name:    "should load system CA",
 			options: TLSConfig{CaCert: ""},
 		},
-		{
-			name:        "should fail with fake system CA",
-			fakeSysPool: true,
-			options:     TLSConfig{CaCert: ""},
-			expectError: "fake system pool",
-		},
```

Comment:
Why is this test removed?

Reply:
Previously the code loaded the system cert pool directly and provided this to `tls.Config`. To support using the system cert pool on Windows, the code now leaves `RootCAs` as `nil` (which has the same effect of causing the `tls` package to use the system cert pool).

Since we aren't loading the SystemCertPool directly, but letting the `tls` package handle this instead, this test isn't relevant anymore.